src/components/table/OcTableFiles.vue

```diff
@@ -388,6 +390,54 @@ export default {
 
       return this.$gettext("Select file")
     },
+
+    getSharedWithAvatarDescription(resource) {
```

Comment:
I'd like to reduce complexity of this. It's sufficient to have `Shared with {user-count} recipients and via {link-count} links`. Don't go into details with files/folders and don't include the names. If it's 200 names I don't want to hear all of them. ;-)

There is still complexity left. We need to:
- a) have proper pluralization (1 recipient / n recipients, 1 link / n links)
- b) shares, links, or both being present

Please look at `$ngettext` and `$gettextInterpolate` for solving this.

Reply:
Could you check my updated version below?

Reply:
@kulmann do we need to add support for other shareTypes than user and link? Like

```js
const groupCount
const guestCount
const remoteCount
```

Reply:
Line 398/399 in this current file, seems a bit to simplistic from my POV

Reply:
If we/you implement additional distinctions here, please also keep the other computed properties in mind 🙂 
 https://github.com/owncloud/owncloud-design-system/blob/master/src/components/avatars/OcAvatarGroup.vue#L112

Reply:
The share types currently are `link` and `everything else`. That is fine. We can't name it `user` because of that.